### ISSUE TITLE
Feature request: listing exercises that match a set of statuses

### ISSUE THREAD

**Issue by kytrinyx** [exercism/configlet#688]
The TL;DR is that I'd like a standardized way of listing out all exercises that are `beta` or `active` so that CI test scripts can ignore exercises that are deprecated or `wip`.

Right now I don't think CI has a default/standard tool that lets us process JSON, but all tracks have Configlet on CI.

If we had a way of getting just a list of active exercises, then the tracks could use that as input to ignore `wip` (and deprecated) exercises, even if it's simply using xargs and running the test command for each exercise.

The larger use case that is motivating this request is that I'd like to:

- Use the new `configlet create` (or whatever) command to stub in a bunch of new (incomplete) exercises in a track. They would have status `wip`.
- Use a generator to generate test suites based on the `problem-specifications`
- Open a PR and merge to `main` without breaking CI.

You could still run the tests locally by doing whatever tracks do to run the tests for just one exercise, but CI wouldn’t fail as long as the exercise is `wip`.

Thoughts?

**Comment by ee7** [comment]
Could be handled by `configlet info`.

I'll reply more later, but I'll just quickly point out that for:

> Right now I don't think CI has a default/standard tool that lets us process JSON

`jq` is always installed in the GitHub Actions environment.

Sources of truth:

- [`ubuntu-22.04`](https://github.com/actions/runner-images/blob/ab0d910edc25c43ee6b2d71a25444f34f7c1574a/images/linux/Ubuntu2204-Readme.md#tools)
- [`macos-12`](https://github.com/actions/runner-images/blob/ab0d910edc25c43ee6b2d71a25444f34f7c1574a/images/macos/macos-12-Readme.md#utilities)
- [`windows-2022`](https://github.com/actions/runner-images/blob/ab0d910edc25c43ee6b2d71a25444f34f7c1574a/images/win/Windows2022-Readme.md#tools)

**Comment by ee7** [comment]
As a starter, the below gets the practice exercise slugs that lack the `status` key. Just needs to include `status` of `beta` or explicit `active` too. 

```console
$ jq -r '.exercises.practice | .[] | select(.status == null).slug' < config.json
hello-world
two-fer
leap
bob
allergies
sum-of-multiples
grade-school
hamming
pangram
acronym
isogram
[...]
```

**Comment by kytrinyx** [comment]
Oh, if we have jq, that solves it for me, I think!

**Comment by ee7** [comment]
It could still be nice to have a command like:

```console
$ configlet list --help
Usage:
  configlet list [--concept-exercises | --practice-exercises | --concepts]
                 [--status <status>]

  <status>: comma-separated list of active,beta,deprecated,wip (default: active,beta)
```

So instead of using `jq` we could write something like:

```shell
configlet list --practice-exercises
```

But not a high priority for now, I think.

**Comment by kytrinyx** [comment]
Agreed, definitely a nice to have, but having `jq` unblocks my use case. 👍
